ironfish/src/rpc/routes/wallet/utils.ts

```diff
@@ -42,6 +42,7 @@ export async function serializeRpcAccountTransaction(
   account: Account,
   transaction: TransactionValue,
   _confirmations: number | undefined,
+  _returnSerialized = false,
```

Comment:
nit - the underscored fields look a little odd to me

we often use an 'options' pattern for things like this: 
```
options?: {
  confirmations: number | undefined.
  serialized: boolean
}
```

Reply:
i like that, will add it